censys/cli/utils.py

```diff
@@ -86,9 +86,8 @@ def _write_screen(search_results: Results):  # pragma: no cover
 
 def write_file(
     results_list: Results,
-    file_format: str = "screen",
+    file_format: Optional[str] = None,
     file_path: Optional[str] = None,
```

Comment:
It looks like Mypy is [complaining](https://github.com/censys/censys-python/runs/5455930843?check_suite_focus=true) since we removed the base name, `file_path` probably shouldn't be optional.
```suggestion
    file_path: str,
    file_format: Optional[str] = None,
```